Source/WebCore/inspector/InspectorCanvas.cpp

```diff
@@ -610,35 +610,35 @@ std::optional<InspectorCanvasCallTracer::ProcessedArgument> InspectorCanvas::pro
 {
     if (!argument)
         return std::nullopt;
-    return {{ JSON::Value::create(0), RecordingSwizzleType::WebGLBuffer }};
+    return {{ JSON::Value::create(static_cast<int>(argument->object())), RecordingSwizzleType::WebGLBuffer }};
```

Comment:
Style checker complains about the `{{` but I kept the existing style instead of reformatting.